provisioning/buffer/kubernetes/templates/api/deployment.yaml

```diff
@@ -0,0 +1,120 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: buffer-api
+  namespace: $NAMESPACE
+  labels:
+    app: buffer-api
+    releaseId: $RELEASE_ID
+spec:
+  replicas: $BUFFER_API_REPLICAS
+  selector:
+    matchLabels:
+      app: buffer-api
+  template:
+    metadata:
+      labels:
+        app: buffer-api
+        releaseId: $RELEASE_ID
+        buffer-etcd-client: "true"
+        tags.datadoghq.com/env: "$KEBOOLA_STACK"
+        tags.datadoghq.com/service: "buffer-api"
+        tags.datadoghq.com/version: "$RELEASE_ID"
+      annotations:
+        log: "true"
+        ad.datadoghq.com/tags: '{"git.commit.sha": "$COMMIT_HASH", "git.repository_url": "https://github.com/keboola/keboola-as-code"}'
+    spec:
+      terminationGracePeriodSeconds: 600
+      containers:
+        - name: buffer-api
+          image: $BUFFER_API_REPOSITORY:$BUFFER_API_IMAGE_TAG
+          resources:
+            requests:
+              cpu: "200m"
```

Comment:
Tady si nejsem jisty jestli nedat treba 100, trochu bojujeme s tim ze mene vytizene stacky maji pak malou celkovou utilizaci. Klidne to zatim ale nechme takhle pro worker i api a muzeme to pak zkusit na testingu nejak zatizit jak to bude funkcni abysme videli jak se to chova a jak to pak pripadne skalovat.

Reply:
ok, vyhodnotime to